bulbs/ads/static/js/mobile-ad-placer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var _ = require('lodash/lodash');
+var adHtml = '<div class="dfp dfp-slot-inread" data-ad-unit="inread"></div>';
+
+
+var MobileAdPlacer = {
+  placeAds: function () {
+    var body = $('body');
+    var paragraphs = body.find('p');
+    var wordCount = 0;
+    var adsPlaced = 0;
+
+    for (var i = 0; i < paragraphs.length; i++) {
```

Comment:
ahh makes sense. Unfortunately, forEach doesn't work here because paragraphs is an HTML collection, so I'm going to leave the for loop in... #jsCraziness